hudi-common/src/main/java/org/apache/hudi/client/transaction/lock/InProcessLockProvider.java

```diff
@@ -19,16 +19,15 @@
 
 package org.apache.hudi.client.transaction.lock;
 
+import org.apache.hudi.common.config.HoodieCommonConfig;
 import org.apache.hudi.common.config.LockConfiguration;
 import org.apache.hudi.common.config.TypedProperties;
 import org.apache.hudi.common.lock.LockProvider;
 import org.apache.hudi.common.lock.LockState;
 import org.apache.hudi.common.util.ValidationUtils;
-import org.apache.hudi.config.HoodieWriteConfig;
 import org.apache.hudi.exception.HoodieLockException;
 
 import org.apache.hadoop.conf.Configuration;
-import org.jetbrains.annotations.NotNull;
```

Comment:
Remove import `NotNull` here since hudi-common doesn't include dep `org.jetbrains.annotations`, which could meet ClassNotFound issue when compiling. Also, I see other lockProviders also don't use this annotation, like `ZookeeperBasedLockProvider` and `HiveMetastoreBasedLockProvider`, so simply remove it here.